.eslintrc

```diff
@@ -85,7 +85,7 @@
 		"init-declarations": "off",
 		"no-shadow": "off",
 		"no-label-var": "error",
-		"no-restricted-globals": ["error", "Proxy", "Reflect", "Symbol", "WeakSet"],
+		"no-restricted-globals": ["error", "Reflect", "Symbol", "WeakSet"],
```

Comment:
FWIW, banning ``Reflect`` but not ``Proxy`` makes zero sense.

Reply:
Oh, right